submissions/dummy_submission/the-nuclear-options.md

```diff
@@ -0,0 +1,150 @@
+
+Since the reactors produce their power indirectly, in the form of heat, we need to harness this power using heat exchangers. Each heat exchanger consumes up to 10 MW in terms of heat, and it is best to have enough of them to harness the reactors' maximum output. Hence, the number of heat exchangers needed is the total power output in MW divided by 10. So, in our example we have 1040 MW / 10 MW = 104 heat exchangers.
+
+Heat exchangers use the heat they absorb to convert up to 103.09 units of 15°C water into 500°C steam, every second. The exchangers have to be supplied with water from offshore pumps, which produce at most 1200 water per second. Hence one offshore pump supports `1200/103.09` heat exchangers, or about 11.64 of them. Therefore you can find how many offshore pumps a group of heat exchangers needs by dividing their count by 11.64. Looking at our example, if all the heat exchangers are connected to the same water network, the network would need `104/11.64 = 8.93`, or 9 offshore pumps in total. However, in the example reactor there are 8 water networks of 13 heat exchangers. Hence each group needs `13/11.64 = 1.12`, so a little more than 1 offshore pump each. I went for 2 each, meaning `2x8 = 16` in total.
+
+Moving on from producing steam to turning it into power, every heat exchanger produces 103.09 steam per second while every steam turbine consumes 60 steam per second. Hence the ratio for them is `103.09/60 = 1.718` turbines for every exchanger. If there is a single steam network across our example plant, it would need `104x1.718 = 178.672`, (so 179) turbines. However, the example actually has 12 steam networks with either 7 or 12 heat exchangers, which respectively would need `7x1.718 = 12.026` (so 12, because the 13th turbine would have almost no utilization) or `12x1.718 = 20.616` (so 21) turbines.
+
+A popular simple alternative for steam turbine placement is attaching two to each heat exchanger without connecting anything with pipes. This provides an excess steam consumption capacity that remains unused unless your plant has steam storage (discussed below).
+
+And so all the component counts are covered. If you want to learn more about the mathematics involved here, or if you want a calculator to do most of it for you, the [Factorio Cheat Sheet](https://factoriocheatsheet.com/#nuclear-power) is the resource I'd recommend.
+
+#### What do we consider when placing the components?
+
+In terms of where to build the nuclear plant as a whole, it is quite beneficial to build near or on water, because the plant needs tons of it. Being close to uranium is optional because it can be easily transported via trains.
```

Comment:
```suggestion
In terms of where to build the nuclear plant as a whole, it is quite beneficial to build near or on water, because the plant needs tons of it. Being close to uranium is optional, because it can be easily transported via trains.
```